### ISSUE TITLE
Problem with questions with image

### ISSUE THREAD

**Issue by jovit** [jovit/fy-moodle#19]
On questions with image, the image id is generated, that creates problems when finding a question again

**Comment by jovit** [comment]
Example:
```html
<img src="http://www.ggte.unicamp.br/eam/pluginfile.php/234145/question/questiontext/612815/10/796114/cap11aq039fig1.png" width="102" height="95">
```
```html
<img src="http://www.ggte.unicamp.br/eam/pluginfile.php/234145/question/questiontext/612824/10/796114/cap11aq039fig1.png" width="102" height="95">
```

these two are the same image but the number in the first is different in the second
612815 and 612824

we need to remove this before generating the hash

**Comment by jovit** [comment]
I suggest using a similar approach to the latex codes, using a regex to remove all of the occurrences

**Comment by jovit** [comment]
questiontext/[0-9]*/[0-9]*/[0-9]*/

**Comment by lnarita** [comment]
If you want to use Regex you can remove the entire URL (except the last bit of the endpoint) with `http.*\/` 

```
let a = '<img src="http://www.ggte.unicamp.br/eam/pluginfile.php/234145/question/questiontext/612815/10/796114/cap11aq039fig1.png" width="102" height="95">'
a.replace(/http.*\//g, '')

// <img src="cap11aq039fig1.png" width="102" height="95">
```

Or you can replace all sequence of digits preceded by an slash with `\/\d+`

```
let a = '<img src="https://www.ggte.unicamp.br/eam/pluginfile.php/234145/question/questiontext/612815/10/796114/cap11aq039fig1.png" width="102" height="95">'
a.replace(/\/\d+/g, '')

// <img src="https://www.ggte.unicamp.br/eam/pluginfile.php/question/questiontext/cap11aq039fig1.png" width="102" height="95">
```

`questiontext\/[0-9]*\/[0-9]*\/[0-9]*\/` is also fine

**Comment by jovit** [comment]
Fix merged, going to be released on version `1.4`
